services/chaospod_test.go

```diff
@@ -811,70 +806,6 @@ var _ = Describe("Chaos Pod Service", func() {
 					Expect(chaosPods[0].Spec.Containers[0].Args).Should(Equal(expectedArgs))
 				})
 
-				Context("with a network cloud spec", func() {
```

Comment:
are we deleting these because they're redundant?

Reply:
Now we don't call cloudServicesProvidersManager in chaos_pod so we have to remove it in this unit test here specifically.
I don't know how I can make it work to unit test it, since we call it in the disruption_controller now... Instead I wanted to only rely on e2e-tests 😬...